bika/lims/browser/reports/administration_usershistory.py

```diff
@@ -151,6 +151,7 @@ def __call__(self):
                     if (user == '' or meta['principal'] == user):
                         actorfullname = userfullname == '' and \
                             mt.getMemberById(user) or userfullname
+                        action_date = api.to_date(meta['timestamp'], DateTime())
```

Comment:
I think we should not use the current date as default fallback here, because it covers bugs during this conversion step.